Laika.gmx/objects/obj_laika.object.gmx

```diff
@@ -35,41 +36,32 @@ dest_y = y;
     <event eventtype="3" enumb="0">
       <action>
         <libid>1</libid>
-        <id>601</id>
-        <kind>0</kind>
+        <id>603</id>
+        <kind>7</kind>
         <userelative>0</userelative>
         <isquestion>0</isquestion>
         <useapplyto>-1</useapplyto>
-        <exetype>1</exetype>
-        <functionname>action_execute_script</functionname>
+        <exetype>2</exetype>
+        <functionname></functionname>
         <codestring></codestring>
         <whoName>self</whoName>
         <relative>0</relative>
         <isnot>0</isnot>
         <arguments>
           <argument>
-            <kind>9</kind>
-            <script>scr_laika_movement</script>
-          </argument>
-          <argument>
-            <kind>0</kind>
-            <string></string>
-          </argument>
-          <argument>
-            <kind>0</kind>
-            <string></string>
-          </argument>
-          <argument>
-            <kind>0</kind>
-            <string></string>
-          </argument>
-          <argument>
-            <kind>0</kind>
-            <string></string>
-          </argument>
-          <argument>
-            <kind>0</kind>
-            <string></string>
+            <kind>1</kind>
+            <string>var _tempSpeed
+if(mouse_check_button(mb_right)) {
+    direction=point_direction(x,y,mouse_x,mouse_y);
+    dist=point_distance(x,y,mouse_x,mouse_y);
+}
+if(dist&gt;0) {
```

Comment:
`(dist>0)`
`(dist > 0)`

Also, if you are checking the distance to zero, won't it just vibrate around the destination, since it never really reaches it?